### PR TITLE
DPC-234: Delete client tokens from web app

### PR DESCRIPTION
**Pull request by Sun-Mountain**
### Fixes [DPC-234](https://jira.cms.gov/browse/DPC-234)

As a user, I need the ability to remove one of my tokens so that I can rotate them or remove a lost token.

This will be on the website after login.

An API delete function already exists to do this but there is no UI.

### Proposed Changes

- Client token delete function added.
- UI for web app
- Response styling fixes

### Change Details

- UI
Screen Shot 2020-08-05 at 12 29 18 PM
Screen Shot 2020-08-05 at 12 29 25 PM

- Responsive styling fixes
Screen Shot 2020-08-05 at 12 30 24 PM
Screen Shot 2020-08-05 at 12 30 33 PM

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [x] no PHI/PII is affected by this change

### CODE REVIEW

File: dpc-web/app/services/api_client.rb [hunk 1]
```diff
@@ -71,7 +77,7 @@ def get_public_keys(reg_org_api_id)
   end
 
   def response_successful?
-    @response_status == 200
+    [200, 204].include? @response_status
```
Comment (dhgreene) [inline dpc-web/app/services/api_client.rb:80]: Good.  Better would be to allow anything in the 20x range--201 and 202 are likely to be used, for instance.

Comment (Sun-Mountain) [reply dpc-web/app/services/api_client.rb:80]: @SMLuthi

Comment (SMLuthi) [reply dpc-web/app/services/api_client.rb:80]: @dhgreene @Sun-Mountain in that case we can do something like `(200..299).include? @response_status` which checks over a range

File: dpc-web/app/controllers/client_tokens_controller.rb [hunk 2]
```diff
@@ -6,42 +6,60 @@ class ClientTokensController < ApplicationController
...
   def create
     @organization = current_user.organizations.find(params[:organization_id])
 
     reg_org = @organization.registered_organization
     manager = ClientTokenManager.new(registered_organization: reg_org)
-    if missing_params
-      render_error 'Label required.'
-    elsif manager.create_client_token(label: params[:label])
+
+    if params_present? && manager.create_client_token(label: params[:label])
```
Comment (SMLuthi) [inline dpc-web/app/controllers/client_tokens_controller.rb:18]: Would this be a little cleaner?

```
return render_error 'Label required.' unless params_present?

if manager.create_client_token(label: params[:label])
  @client_token = manager.client_token
  render :show

render_error 'Client token could not be created.'
```

Comment (Sun-Mountain) [reply dpc-web/app/controllers/client_tokens_controller.rb:18]: The tests did not like that >_>

Comment (SMLuthi) [reply dpc-web/app/controllers/client_tokens_controller.rb:18]: Hm, that shouldnt happen. I wonder if the render_errors arent being hit correctly in the way you defined them.

**Pull request merged**